src/crc32c_unittest.cc

```diff
@@ -83,7 +83,7 @@ TEST(CRC32CTest, Crc32cStdString) {
   EXPECT_EQ(static_cast<uint32_t>(0x8a9136aa), crc32c::Crc32c(buf));
 
   for (size_t i = 0; i < 32; ++i)
-    buf[i] = static_cast<char>(0xff);
+    buf[i] = static_cast<char>(-1);  // 0xff causes warning C4309 on MSVC.
```

Comment:
What about?

```c
buf[i] = '\xff'
```

Reply:
Done.

Sweet, I didn't think about that! Thanks!